recipes/phylip/tests/Makefile

```diff
@@ -0,0 +1,45 @@
+# Makefile, tebsts directory and samples by Catherine Letondal <letondal@pasteur.fr>
+
```

Comment:
We do not include test files in bioconda. Functional tests should be done upstream. Could you simply add something similar to `phylib --help` or --version?